torchaudio/kaldi_io.py

```diff
@@ -63,16 +63,15 @@ def read_vec_int_ark(file_or_fd):
 
 
 def read_vec_flt_scp(file_or_fd):
-    r"""Create generator of (key,vector<float32/float64>) tuples, read according to kaldi scp.
+    r"""Create generator of (key,vector<float32/float64>) tuples, read according to Kaldi scp.
 
-    Arguments:
-        file_or_fd (string/File Descriptor): scp, gzipped scp, pipe or opened file descriptor.
+    Args:
+        file_or_fd (str/FileDescriptor): Scp, gzipped scp, pipe or opened file descriptor
```

Comment:
lower case scp, since it's a command?

Reply:
Hmm I'm not sure about the correct grammar here. scp and ark are both file extensions and so at the beginning of sentence would you put "Mp3 ...." or "mp3 ..."?

https://kaldi-asr.org/doc/io.html